frontends/pytorch/examples/torchscript_mm_e2e.py

```diff
@@ -0,0 +1,51 @@
+# -*- Python -*-
+# This file is licensed under a pytorch-style license
+# See frontends/pytorch/LICENSE for license information.
+
+import typing
+
+import torch
+import torch_mlir
+
+import npcomp
+from npcomp.compiler.pytorch.backend import refjit, frontend_lowering
+from npcomp.compiler.utils import logging
+
+import test_utils
+
+logging.enable()
+
+# RUN: %PYTHON %s | npcomp-opt | FileCheck %s
+
+mb = torch_mlir.ModuleBuilder()
+
+class TestModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    def forward(self, lhs, rhs):
+        return torch.mm(lhs, rhs)
+
+test_module = TestModule()
+class_annotator = torch_mlir.ClassAnnotator()
+recursivescriptmodule = torch.jit.script(test_module)
+torch.jit.save(recursivescriptmodule, '/tmp/foo.pt')
+
+class_annotator.exportNone(recursivescriptmodule._c._type())
+class_annotator.exportPath(recursivescriptmodule._c._type(), ['forward'])
+class_annotator.annotateShapesAndDtypes(recursivescriptmodule._c._type(), ['forward'], [
```

Comment:
Do this a few more times and you will tempt me in to rage-prettifying this :)

Reply:
Don't worry, I already have the new pretty python stuff working locally. Will send PR soon. As a preview, it looks liek this:

```
from torch_mlir.torchscript.annotations import annotate_args, export

class MmModule(torch.nn.Module):
    def __init__(self):
        super().__init__()
    @export
    @annotate_args([
        None,
        ([-1, -1], torch.float32),
        ([-1, -1], torch.float32),
    ])
    def forward(self, lhs, rhs):
        return torch.mm(lhs, rhs)
```